json-2/src/main/java/com/baeldung/jsonld/deserialization/jsonldjava/jackson/JacksonDeserializationExample.java

```diff
@@ -0,0 +1,40 @@
+package com.baeldung.jsonld.deserialization.jsonldjava.jackson;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.jsonldjava.core.JsonLdOptions;
+import com.github.jsonldjava.core.JsonLdProcessor;
+import com.github.jsonldjava.utils.JsonUtils;
+
+public class JacksonDeserializationExample {
+    public static void main(String[] args) throws IOException {
+        String exampleJsonld ="{" + 
+           "  \"@context\": {" + 
+           "    \"@vocab\": \"http://schema.org/\"," + 
+           "    \"knows\": {" + 
+           "      \"@type\": \"@id\"" + 
+           "    }" + 
+           "  }," + 
+           "  \"@type\": \"Person\"," + 
+           "  \"@id\": \"http://example.com/person/1234\"," + 
+           "  \"name\": \"Example Name\"," + 
+           "  \"knows\": \"http://example.com/person/2345\"" + 
+           "}";
+
+        Object jsonObject = JsonUtils.fromString(exampleJsonld);
+        Object compact = JsonLdProcessor.compact(jsonObject, new HashMap(), new JsonLdOptions());
```

Comment:
Let's fix the warning about raw types if possible.

Reply:
I must have some settings not properly set because I see no warnings. Could you point me in the right direction?

Reply:
The warning here is - HashMap is a raw type. References to generic type HashMap<K,V> should be parameterized. It can be fixed like - new HashMap<>()